contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -49,8 +49,7 @@ mv temp-go/go/* /root/.go/
 cd $SRC/containerd
 
 go mod tidy
-rm vendor/github.com/cilium/ebpf/internal/btf/fuzz.go
-rm /root/go/pkg/mod/github.com/cilium/ebpf@v0.7.0/internal/btf/fuzz.go
+go get github.com/cilium/ebpf@latest
```

Comment:
Should this get a specific version so it doesn't break later?

Reply:
+1